interfaces/builtin/system_observe.go

```diff
@@ -123,9 +130,12 @@ ptrace (read),
 /sys/fs/cgroup/cpu,cpuacct/cpu.stat r,
 /sys/fs/cgroup/memory/memory.stat r,
 
-# Allow reading the system max CPU resource constraints 
+# Allow reading the system max CPU resource constraints
 /sys/fs/cgroup/system.slice/cpu.max r,
 
+# Allow reading Btrfs filesystem information
+/sys/fs/btrfs/* r,
```

Comment:
This permission is granted since the `btrfs` node-exporter collector is trying to access the files in this directory, see:

- [Ref1](https://github.com/prometheus/node_exporter/blob/434fb7d864fe960f67681fdcf973e48aecf4a0d7/collector/btrfs_linux.go#L28)
- [Ref2](https://github.com/prometheus/procfs/blob/53e9e6772ade719daba4f16d7d9f007997b532a3/btrfs/get.go#L57C38-L57C40)